data/templates/set_bmc_credentials.sh

```diff
@@ -47,14 +49,38 @@ for x in $cmdReturn; do
   i=$((i+1))
 done
 
+function get_newUserNumber()
+{
+        cmdReturn=$(sudo ipmitool user summary $channel)
+        myarray=(${cmdReturn//$'\n'/ })
+        MaxCount=${myarray[3]}
```

Comment:
Still a naming style suggestion, previously all variable name uses the camel case format (begins with lower letter capital), keep this style will be better. 

The same for one additional variable `ID` below.